script/fake_data.py

```diff
@@ -20,13 +20,27 @@
     ForecastValueSQL,
 )
 from nowcasting_datamodel.models.models import StatusSQL
+from sqlalchemy import inspect
 
 from src.utils import floor_30_minutes_dt
 
 now = floor_30_minutes_dt(datetime.now(tz=timezone.utc))
 
+if os.getenv("DB_URL") is None:
+    raise ValueError("Please set the DB_URL environment variable")
+
+if not os.getenv("DB_URL") == "postgresql://postgres:postgres@postgres:5432/postgres":
+    raise ValueError("This script should only be run in the local docker container")
+
 connection = DatabaseConnection(url=os.getenv("DB_URL", "not_set"))
 
+print(f"Creating fake data at {now}")
+
+# Check database has been created, if not run the migrations
+if not inspect(connection.engine).has_table("status"):
+    connection.create_all()
```

Comment:
Could you add a warning print statement in here, saying I'm making SQL tables. Just so its clear for people

Reply:
If it's about to create ~10+ tables, so any print statements here will be gone immediately... if we actually want to warn the user before creating the tables, we'd have to change how this is set up